framework-plugins/lisk-framework-forger-plugin/test/utils/application.ts

```diff
@@ -14,24 +14,43 @@
 import * as os from 'os';
 import * as fs from 'fs-extra';
 import * as path from 'path';
-import { Application, ApplicationConfig, GenesisBlockJSON } from 'lisk-framework';
-import * as genesisBlockJSON from '../fixtures/genesis_block.json';
+import { Application, ApplicationConfig } from 'lisk-framework';
+import { validator } from '@liskhq/lisk-validator';
 import * as configJSON from '../fixtures/config.json';
 import { ForgerPlugin } from '../../src';
+import { getGenesisBlockJSON } from './genesis_block';
 
 const forgerApiPort = 5001;
 
+export const startApplication = async (app: Application): Promise<void> => {
+	// TODO: Need to figure out why below error appears but its only in tests
+	//  Trace: Error: schema with key or id "/block/header"
+	validator.removeSchema('/block/header');
```

Comment:
Hope you have a issue for this if its not covered in this PR. otherwise we might forget the TODO.

Reply:
https://github.com/LiskHQ/lisk-sdk/issues/5572